lib/cats/basemeow.rb

```diff
@@ -0,0 +1,21 @@
+module Cats
+  class Basemeow
+    attr_accessor :times,:meow
+    def initialize (times, meow)
+      @times = times
+      @meow = meow
+    end
+
+    def to_boolean(val)
+      if val && val =~ /^(true|t|yes|y|1)$/i
+        true
+      else
+        false
```

Comment:
Technically speaking, you can drop the `else false` here because the method has a false-y default return value.

```ruby
def to_boolean(val)
      if val && val =~ /^(true|t|yes|y|1)$/i
          true
      end
end
```

or if you wanted to be really brief, you can omit the conditional entirely

```ruby
def to_boolean(val)
      val && val =~ /^(true|t|yes|y|1)$/i
end
```